src/poetry_stickywheel_plugin.py

```diff
@@ -72,21 +95,23 @@ def update_dependency_group(
     def pin_dependency(
         self, dependency: poetry.core.packages.directory_dependency.DirectoryDependency
     ) -> poetry.core.packages.dependency.Dependency:
-        pyproject_file = dependency.path / "pyproject.toml"
+        dep_pyproject_file = dependency.path / "pyproject.toml"
```

Comment:
can we rename this back to reduce the number of lines changed?

otherwise it looks pretty solid to me.

Reply:
Sure!

Reply:
Done :slightly_smiling_face: